styles.css

```diff
@@ -288,3 +282,143 @@ li {
   color: #253858;
   margin-left: 20px;
 }
+
+.form {
+  background-color: #6070ff;
+  background-image: url("./contact\ form\ shapes-mobile.svg");
+  background-position: right;
+  background-repeat: no-repeat;
+  margin-top: 40%;
+  border-top-left-radius: 10%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓